client/web/src/auth/welcome/StartSearching.tsx

```diff
@@ -129,15 +129,15 @@ export const StartSearching: React.FunctionComponent<StartSearching> = ({
 
     return (
         <div className="mt-5">
-            <h3>Start searching...</h3>
+            <h4>Fetching repositories...</h4>
```

Comment:
```suggestion
            <h3>Fetching repositories...</h3>
```

Sorry, we got swapped somewhere!